<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tableindex_widget</class>
 <widget class="QWidget" name="tableindex_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>61</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
